src/main/java/org/breedinginsight/brapi/v2/services/BrAPIGermplasmService.java

```diff
@@ -138,14 +138,14 @@ public List<Map<String, Object>> processListData(List<BrAPIGermplasm> germplasm,
                 }
             }
 
-            if ((germplasmEntry.getPedigree() != null) && (!germplasmEntry.getPedigree().isEmpty())) {
-                Pedigree germPedigree = Pedigree.parsePedigreeString(germplasmEntry.getPedigree());
-                if (!germPedigree.maleParent.isEmpty()) {
-                    row.put("Male Parent GID", Integer.parseInt(germPedigree.maleParent));
-                }
-                if (!germPedigree.femaleParent.isEmpty()) {
-                    row.put("Female Parent GID", Integer.parseInt(germPedigree.femaleParent));
-                }
+            if (germplasmEntry.getAdditionalInfo().get(BrAPIAdditionalInfoFields.GERMPLASM_MALE_PARENT_GID) != null) {
```

Comment:
In the latest spec it's specified to use PURDY pedigree notation strings so it can specify relationships using different delimiters. In some past work it made more sense to store the GIDs in additionalinfo to just read the straight values out rather than doing a bunch of processing. So I just updated this bit of code to take advantage of that work.

Reply:
Got it, thanks.